contrib/buildrefactor/src/python/pants/contrib/buildrefactor/buildozer.py

```diff
@@ -52,12 +53,18 @@ def execute(self):
     if self.options.remove_dependencies:
       self.remove_dependencies()
 
+    if self.options.command:
+      self.execute_custom_command()
+
```

Comment:
From UX perspective, if both `--command` and (`--remove-dependencies` or `add-dependencies`) are specified, this may be unclear what the behaviors is.

Let me know if this makes sense, i.e. we prioritize custom command like:
```
if self.options.command:
  if self.options.remove_dependencies or self.options.add_dependencies: # with potentially more options
    raise an error.
  self.execute_custom_command()
```

Reply:
I like the idea of prioritizing the custom command and raising an error if the custom command is used in combination with `remove_dependencies` and/or `add_dependencies`. 

Would it be more desirable to have a warning or an error? I could see someone using a combination of the options, even if that might be weird.

Reply:
I'd say error out to avoid any weird behavior.